src/kernel/chainparams.cpp

```diff
@@ -250,7 +250,7 @@ class CTestNetParams : public CChainParams {
         nDefaultPort = 18333;
         nPruneAfterHeight = 1000;
         m_assumed_blockchain_size = 42;
-        m_assumed_chain_state_size = 3;
+        m_assumed_chain_state_size = 11;
```

Comment:
985d1ec267f44b7897855fd2312da76b3fcd165b: this is already 13G for me.

Reply:
It's 9.9G for me. Are you looking at the right network datadir?

Reply:
I see similar size to @Sjors, as of height=2873716:
```
du -sh ~/.bitcoin/testnet3/chainstate/ ~/.bitcoin/testnet3/blocks/
13G	/home/martin/.bitcoin/testnet3/chainstate/
82G	/home/martin/.bitcoin/testnet3/blocks/
```
(I wasn't continuously online during the fork shenanigans, so having fewer blocks makes sense).

Reply:
Either way, `m_assumed_blockchain_size` can be updated here too.

Reply:
```
$ du -sh blocks/ chainstate/
84G	blocks/
13G	chainstate/
```

Reply:
I'm quite confused as to how all of you have way larger blocks and chainstate dirs than I do. I checked my 3 nodes and they all agree on these sizes, and all 3 have very high uptimes.

Anyways, bumped `m_assumed_blockchain_size` to 93 and `m_assumed_chain_state_size` to 19, which I think should cover all sizes given here.